distribution/client/package.json

```diff
@@ -7,7 +7,8 @@
     "client": "node src/client.js",
     "test": "npm run eslint && npm run jest",
     "eslint": "eslint src/. test/. --config .eslintrc.json",
-    "jest": "jest"
+    "jest": "jest",
+    "prepare": "patch-package"
```

Comment:
This should be executed in the `Makefile` shouldn't it?

Reply:
It's done automatically. Because the patching is bound to the standard 'prepare' script. 

BTW that is also why I added the associated test: to check and show this does get applied.